geomstats/geometry/full_rank_matrices.py

```diff
@@ -0,0 +1,108 @@
+"""Module exposing the full rank euclidean matrices"""
+
+import geomstats.backend as gs
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.matrices import Matrices
+from geomstats.geometry.matrices import MatricesMetric
+
+
+class FullRankMatrices(OpenSet):
+    """Class for the matrices with euclidean entries and full rank.
```

Comment:
What do you mean by Euclidean entries?

Reply:
taking values in R. I change it to Class for R_*(m,n) matrices of dimension m x n and full rank.